### PR TITLE
Add guidance on open formats to attachments admin pages

### PR DESCRIPTION
**Pull request by barrucadu**
screen shot 2018-04-10 at 14 44 21
screen shot 2018-04-10 at 14 44 23
screen shot 2018-04-10 at 14 44 24

I used an `alert-warning` on the attachments list page as I think it should stand out, but having two `alert-info`s next to each other doesn't seem very good.

Links go directly to the `gov.uk` domain, rather than through any smart URL constructors, but that seems in keeping with other guidance links:

```
$ git grep www.gov.uk/guidance
...
app/views/admin/dashboard/index.html.erb:      <li><a href="https://www.gov.uk/guidance/style-guide">GOV.UK style guide</a></li>
app/views/admin/dashboard/index.html.erb:      <li><a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk">How to publish content on GOV.UK</a></li>
app/views/admin/dashboard/index.html.erb:      <li><a href="https://www.gov.uk/guidance/content-design">Planning, writing and managing content</a></li>
...
```

### CODE REVIEW

File: app/views/admin/attachments/index.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <p class="qa-helper-copy">
         <strong>Note:</strong>
         <%= attachment_note(attachment.attachable_model_name) %>
-      <p>
+      </p>
```
Comment (thomasleese) [inline app/views/admin/attachments/index.html.erb]: 😬

File: app/views/admin/attachments/index.html.erb [hunk 2]
```diff
@@ -31,6 +31,11 @@
         <% end %>
       </ul>
 
+      <p class="alert alert-warning">
```
Comment (thomasleese) [inline app/views/admin/attachments/index.html.erb]: Could we make this a `div` and then use a `p` for each sentence inside it to leave a small gap between them?

**Comment by kevindew** [comment]
Are we in charge of the wording here? If we use the phrase "must" that the proprietary formats will be rejected but I assume this is just advice.

**Comment by barrucadu** [comment]
I don't know, I copied the wording from the Trello card.  "Must" does seem a bit strong, and isn't the wording used by the linked guidance (which says "is committed").

**Comment by kevindew** [comment]
@barrucadu sure, yeah that sounds like a chat to have with Martin. I'll ping him about it.

**Comment by barrucadu** [comment]
With a div:
screen shot 2018-04-10 at 15 35 22

**Comment by edent** [comment]
From the original guidance https://www.gov.uk/government/publications/open-standards-for-government/sharing-or-collaborating-with-government-documents

> Citizens, businesses and delivery partners **must** be able to interact with government officials and services, or those working on behalf of government, sharing appropriately formatted, editable documents.

**Pull request merged**